help/start-a-call.md

```diff
@@ -15,7 +15,7 @@ compose box.
 
 !!! tip ""
 
-    You can replace "Click to join video call" with anything you want.
+    You can replace "Join video call." with anything you want.
```

Comment:
Let make this:

> You can replace the "Join video call." label for the link with any text you like.

Reply:
Have added the updated help center screenshot to the PR description